sdk/textanalytics/ai-text-analytics/swagger/README.md

```diff
@@ -0,0 +1,417 @@
+# Azure Text Analytics TypeScript Protocol Layer
+
+> see https://aka.ms/autorest
+
+## Configuration
+
+```yaml
+package-name: "@azure/ai-text-analytics"
+title: GeneratedClient
+description: TextAnalytics Client
+generate-metadata: false
+license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: ../
+source-code-folder-path: ./src/generated
+# FHIR PR: https://github.com/Azure/azure-rest-api-specs/pull/17710
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/cognitiveservices-Language-2022-04-01-preview/specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json
+add-credentials: false
+package-version: 6.0.0-beta.1
+v3: true
+hide-clients: true
+typescript: true
+```
+
+## Customizations for Track 2 Generator
+
+See the [AutoRest samples](https://github.com/Azure/autorest/tree/master/Samples/3b-custom-transformations)
+for more about how we're customizing things.
+
+### Update version
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $.info
+    transform: $.version = "2022-03-01-preview";
```

Comment:
this is temporary while 2022-04-01 is available?

Reply:
Yeah this is a quick hack until 2022-04-01 is fully deployed.